.manylinux-install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    if [[ "${PYBIN}" == *"cp27"* ]] || \
+       [[ "${PYBIN}" == *"cp34"* ]] || \
+       [[ "${PYBIN}" == *"cp35"* ]] || \
+       [[ "${PYBIN}" == *"cp36"* ]]; then
+        "${PYBIN}/pip" install -e /io/
+        "${PYBIN}/pip" wheel /io/ -w wheelhouse/
+    rm -rf /io/build /io/*.egg-info
```

Comment:
Indentation seems wrong here: the rm is inside the `if`, but is indented as if it's outside.